anomalib/models/cflow/torch_model.py

```diff
@@ -0,0 +1,130 @@
+"""PyTorch model for CFlow model implementation."""
+
+# Copyright (C) 2020 Intel Corporation
```

Comment:
Minor issue but should be be updating the dates? Is there a standard practice for this?